about.html

```diff
@@ -0,0 +1,240 @@
+                            </a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </section>
+    </header>
+
+    <section class="headline_wrapper">
+        <div class="headline">
+            <div class="title_wrapper">
+                <div class="suport_title hide--desktop">
+                    <p>"Hello! Sharing world"</p>
+                </div>
+                <div class="title">
+                    <h1>
+                        CREATIVE CODERS GLOBAL SUMMIT 2023
+                    </h1>
+                </div>
+            </div>
+        </div>
+
+        <div class="description_wrapper">
+            <div class="description">
+                <p class="hide--desktop">The Global Summit brings together the community of
+                    experts, academics, and activists who comprise the Creative Commons affiliate network in a different
+                    country every two years.
+                    This year, we hope to expand our invitation list including organizations and individuals.</p>
+
+                <p class="hide--mobile">A developer conference is a gathering of software developers, engineers, and
+                    other professionals who work in the field of software development. These conferences are typically
+                    held in large cities around the world and are organized by companies, professional organizations, or
+                    communities of developers. The main goal of a developer conference is to provide a forum for
+                    developers to learn about new technologies, techniques, and best practices in software development.
+                    At a developer conference, attendees can expect to participate in a variety of activities such as
+                    presentations, workshops, panels, and networking events. These events are typically led by industry
+                    experts and thought leaders, who share their knowledge and experience with the attendees. Many
+                    developer conferences also feature exhibits and demos of new products and tools, giving attendees
+                    the opportunity to try out the latest technology firsthand.</p>
+            </div>
+        </div>
+
+        <div class="email_contact_wrapper">
+            <div class="email_contact_container">
+                <p class="text_to_email">Please contact us per Email for nay further questions about CC Global Summit
+                    2015!</p>
+                <a href="#" class="email_contact">ccoderssum2023@cccoders.org ></a>
+            </div>
+        </div>
+    </section>
+
+    <section class="summit_info_wrapper">
+        <div class="summit_info">
+            <div class="summit_title_wrapper">
+                <div class="summit_title_container">
+                    <h2 class="summit_title">Creative Coders <br class="hide--desktop">
+                        Global Summit 2015 Logo</h2>
```

Comment:
- [ ] Did you mean Global Summit **2023**? Please 🙏🏽  correct the year to personalize your page.